README.md

```diff
@@ -1,28 +1,27 @@
 # Introduction
 
-This repository contains all the annotation training data and models used by the PZH Natura 2000 remote sensing project.
-In short for this project we want to use machine learning on land satellite images to monitor each pixel distribution of various nature types in nature protected areas across time for various climate/nature policies.
-So the models here predicts for each pixel in a land satellite image a nature type, these types could include: Grass,Forest,Sand ..etc based on the learned annotations.
+This repository houses all the training data , models and code to make said models utilized in the PZH Natura 2000 remote sensing project. The aim of this project is to employ machine learning techniques on land satellite imagery to monitor the distribution of various nature types within protected areas over time, in alignment with diverse climate/nature policies. Therefore, the models herein predict the nature type for each pixel in a land satellite image, which could include Grass, Forest, Sand, etc., based on learned annotations.
```

Comment:
This part of the text still says the repo houses all training data & models, although you've moved them to a blob storage!

Reply:
technically it does contain them then? maybe not the data it's but links to the blob storage?

Reply:
Sure, fair enough. Especially now that you put the data/models on a publicly available blob storage.